From ea90a79686baf352bb6b4b365317d7b33834a438 Mon Sep 17 00:00:00 2001
From: "ji, zhenlong z" <zhenlong.z.ji@intel.com>
Date: Fri, 16 Jul 2021 15:06:24 +0800
Subject: [PATCH] Disable adb authentication for PGP

Since the looking glass in PGP has dependence on adb, so
we need to enable adb and disable the adb authentication on
PGP's user build.

Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>
---
 adb/daemon/auth.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adb/daemon/auth.cpp b/adb/daemon/auth.cpp
index 1a1e4ad62..cbf4360ec 100644
--- a/adb/daemon/auth.cpp
+++ b/adb/daemon/auth.cpp
@@ -63,7 +63,7 @@ static struct adisconnect adb_disconnect = {adb_disconnected, nullptr};
 static android::base::NoDestructor<std::map<uint32_t, weak_ptr<atransport>>> transports;
 static uint32_t transport_auth_id = 0;
 
-bool auth_required = true;
+bool auth_required = false;
 
 static void* transport_to_callback_arg(atransport* transport) {
     uint32_t id = transport_auth_id++;
-- 
2.25.1

